packages/analytics-js-integrations/src/integrations/Braze/browser.js

```diff
@@ -75,7 +75,17 @@ class Braze {
   }
 
   isReady() {
-    return this.isLoaded();
+    if (this.isLoaded()) {
+      try {
+        const anonymousId = this.analytics.getAnonymousId();
+        window.braze.getUser().addAlias(anonymousId, 'rudder_id');
```

Comment:
Can we use a more appropriate property name here? Like `rudderstack_id`

Reply:
We need to maintain it keep consistency for cloud mode and hybrid mode
ref: https://github.com/rudderlabs/rudder-transformer/blob/3c6e0476e7537574ca150e494096c2ecfde93008/src/v0/destinations/braze/util.js#L609